configure.ac

```diff
@@ -414,4 +418,5 @@ Features enabled
     CBCP.................: ${enable_cbcp:-no}
     IPXCP................: ${enable_ipxcp:-no}
     EAP-TLS..............: ${enable_eaptls:-yes}
+    systemd notifications: ${enable_systemd:-no}
```

Comment:
For consistency with other features above, I may want to merge this as SYSTEMD instead of "systemd notifications"

Reply:
On reflection: why? systemd don't call it that upstream, and they're rather picky about the styling of the [name](https://www.freedesktop.org/wiki/Software/systemd/):
>Yes, it is written systemd, not system D or System D, or even SystemD. And it isn't system d either. Why? Because it's a system daemon, and under Unix/Linux those are in lower case, and get suffixed with a lower case d. And since systemd manages the system, it's called systemd. It's that simple. But then again, if all that appears too simple to you, call it (but never spell it!) System Five Hundred since D is the roman numeral for 500 (this also clarifies the relation to System V, right?). The only situation where we find it OK to use an uppercase letter in the name (but don't like it either) is if you start a sentence with systemd. On high holidays you may also spell it sÿstëmd. But then again, Système D is not an acceptable spelling and something completely different (though kinda fitting).